src/main/scala/org/broadinstitute/hail/driver/AnnotateSamplesExpr.scala

```diff
@@ -28,6 +32,7 @@ object AnnotateSamplesExpr extends Command {
 
   def run(state: State, options: Options): State = {
     val vds = state.vds
+    val branchingFactor = Option(options.branchingFactor).map(x => x: Int)
```

Comment:
The declared type for `options.branchingFactor` is `java.lang.Integer` so the default is `null`.  `Option[Int](options.branchingFactor)` gets a null-pointer exception because the conversion to `Int` happens before the test for `null` in the constructor.  I agree it is gross, happy to consider other ways of writing it.